src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -361,6 +362,9 @@ def acs_create(resource_group_name, deployment_name, name, ssh_key_value, dns_na
      if raw=true
     :raises: :class:`CloudError<msrestazure.azure_exceptions.CloudError>`
     """
+    if ssh_key_value is not None and not _is_valid_ssh_rsa_public_key(ssh_key_value):
+        raise CLIError('Provided ssh key ({}) is invalid or non-existent'.format(ssh_key_value))
```

Comment:
If I pass in the key value (rather than the path) to `--ssh-key-value`, would this print the full public key in the error message?
Is this intended?